core/primitives/src/test_utils.rs

```diff
@@ -46,13 +47,28 @@ pub fn init_test_module_logger(module: &str) {
         .filter_module(module, LevelFilter::Info)
         .filter(None, LevelFilter::Info)
         .try_init();
+    init_stop_on_panic();
 }
 
 pub fn init_integration_logger() {
     let _ = env_logger::Builder::new()
         .filter(None, LevelFilter::Info)
         .filter(Some("actix_web"), LevelFilter::Warn)
         .try_init();
+    init_stop_on_panic();
+}
+
+static SET_PANIC_HOOK: Once = Once::new();
+
+/// This is a workaround to make actix/tokio runtime stop when a task panics.
+pub fn init_stop_on_panic() {
```

Comment:
Doesn't our node stop when one of the actix tasks panic?

Reply:
not with tokio 0.2: now it catches panics